indra/newview/app_settings/settings.xml

```diff
@@ -13829,6 +13829,17 @@
       <key>Value</key>
       <integer>2</integer>
     </map>
+    <key>UIPreviewMaterial</key>
+    <map>
+      <key>Comment</key>
+      <string>Whether or not PBR material swatch is enabled</string>
+      <key>Persist</key>
+      <integer>1</integer>
```

Comment:
probably want this not to persist if we are setting it from simulator features?

Reply:
Nah, persisting is fine.  Will make it default to whatever it last saw, which is fine.